### ISSUE TITLE
Did anyone preproduce pix2pix-turbo ?

### ISSUE THREAD

**Issue by Lucarqi** [GaParmar/img2img-turbo#93]
I want to use pix2pix for a medical image generation task, where the control condition is different label images. My hyperparameter settings are as follows:
accelerate launch src/train_pix2pix_turbo.py \
    --pretrained_model_name_or_path="stabilityai/sd-turbo" \
    --output_dir="output/pix2pix_turbo/test" \
    --dataset_folder="data/data_u3/pix2pix" \
    --resolution=256 \
    --num_training_epochs=500 \
    --train_batch_size=2 \
    --enable_xformers_memory_efficient_attention \
    --viz_freq 25 \
    --track_val_fid \
    --report_to "wandb" \
    --tracker_project_name "test"

accelerate config is:
compute_environment: LOCAL_MACHINE
debug: false
distributed_type: MULTI_GPU
downcast_bf16: 'no'
enable_cpu_affinity: false
gpu_ids: all
machine_rank: 0
main_training_function: main
mixed_precision: 'no'
num_machines: 1
num_processes: 3
rdzv_backend: static
same_network: true
tpu_env: []
tpu_use_cluster: false
tpu_use_sudo: false
use_cpu: false

I found that during training, the output images from the model are always the same and significantly different from the real images.：
![image](https://github.com/user-attachments/assets/3b067755-d257-4a2d-9b0d-04873c44a884)

And during testing, all synthetic images are also almost identical:
![image](https://github.com/user-attachments/assets/f0898abf-fbf3-4723-af2b-9ea32cc40eff)
![image](https://github.com/user-attachments/assets/1285176c-5979-4745-806e-f83031ee1b19)
![image](https://github.com/user-attachments/assets/c79f8fb6-5bbb-45dc-a336-b22e80c335fc)

training wandb:
![image](https://github.com/user-attachments/assets/068208c3-5eb3-4160-bba2-1c10f03ee403)

validation wandb:
![image](https://github.com/user-attachments/assets/02702d1c-78bc-4b84-bc8e-1fef3e4b14ce)

This issue has been consistently troubling me. Does anyone have any unique insights about it?

**Comment by GaParmar** [comment]
Looking at your results I can spot a couple issues with your setting:
1. Since you are doing medical image segmentation, you can probably remove the CLIP_SIM loss. The text-image similarity loss is probably not very useful in this setting. 
2. Maybe your model is underfitting, you can try increasing the rank of the LoRa adapters

**Comment by Lucarqi** [comment]
@GaParmar thanks for your reply. 
For 1: At the beginning, I used the default settings, including clip_sim loss, but it also had the same effect.
For 2: I will try your advise.

**Comment by zmingen** [comment]
@Lucarqi Hi, I also have the same issue. The generated images are almost identical. Have you find a solution?

**Comment by Jamie-Cheung** [comment]
> @Lucarqi Hi, I also have the same issue. The generated images are almost identical. Have you find a solution?

What is the amount of data you train? I have also encountered this problem

**Comment by zmingen** [comment]
@Jamie-Cheung , About 600 image pairs. There is no text prompt in my dataset, maybe this is the root cause.
